### PR TITLE
chore: Increased test coverage for StakingUtiliites 

### LINKED ISSUE #13112: StakingUtilities [68.75 %]
Increase test code coverage for StakingUtilities class

### PR DESCRIPTION
**Pull request by thomas-swirlds-labs**
**Description**:
increased test coverage for StakingUtiliites in Token Service

**Related issue(s)**:

Fixes #13112 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/staking/StakingRewardsHandlerImplTest.java [hunk 1]
```diff
@@ -94,6 +98,28 @@ public void setUp() {
...
+            assertEquals(UnsupportedOperationException.class, cause.getClass());
+        }
+    }
+
+    @Test
+    void testStakeMetaChangesForNullOriginalAccount() {
+        noStakeChanges();
+        final var rewards = subject.applyStakingRewards(context, Collections.emptySet(), emptyMap());
+        final var modifiedAccount = writableAccountStore.get(payerId);
+        assert modifiedAccount != null;
```
Comment (povolev15) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/staking/StakingRewardsHandlerImplTest.java:119]: @thomas-swirlds-labs As I know, we decided to use assertJ that's why  `assertThat(modifiedAccount).isNotNull()` will be more aligned with assertJ. other than that all look good

Comment (mhess-swl) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/staking/StakingRewardsHandlerImplTest.java:119]: +1

Comment (thomas-swirlds-labs) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/staking/StakingRewardsHandlerImplTest.java:119]: Updated. Thanks @povolev15

**Review by povolev15 — APPROVED** [review]
@thomas-swirlds-labs TY LGTM

**Pull request merged**